webpack.embedding-sdk.config.js

```diff
@@ -118,7 +118,7 @@ module.exports = env => {
     },
 
     externals: {
-      ...mainConfig.externals,
+      // ...mainConfig.externals,
```

Comment:
this was casuing a few deps not being bundled -> they were needed to be added as deps.
i'd keep it simple for now, even though eventually we should mark them all as external and not bundle them